course/src/main/java/org/openedx/course/presentation/container/CourseContainerFragment.kt

```diff
@@ -115,6 +121,7 @@ class CourseContainerFragment : Fragment(R.layout.fragment_course_container) {
             addFragment(CourseOutlineFragment.newInstance(viewModel.courseId, courseTitle))
             addFragment(CourseVideosFragment.newInstance(viewModel.courseId, courseTitle))
             addFragment(DiscussionTopicsFragment.newInstance(viewModel.courseId, courseTitle))
+            addFragment(CourseDatesFragment.newInstance(viewModel.courseId, courseTitle))
```

Comment:
We need to update the `binding.viewPager.offscreenPageLimit` as well.

Reply:
any recommendation?

Reply:
replaced with `binding.viewPager.offscreenPageLimit = adapter?.itemCount ?: 1` and moved after the adapter initialization.